test/functionalTest/cases/0000_json_parse/jsonParseBadVerbDiscoverContextAvailability.test

```diff
@@ -25,11 +25,17 @@ JSON Parse BadVerb DiscoverContextAvailability
 
```

Comment:
No actual changes in this file? Only cleanup?

Reply:
Just transforming it to the "new" way

NTC